### DISCUSSION TITLE
using ne5532 or tl072 for front end

### DISCUSSION THREAD

**Discussion by santlaha** [fhdm-dev/scoppy#18] [Scoppy Front end designs]
ne 5532 has many advantages.
1. easily available for use in audio sector
2. 10 mhz bandwith 
3. low noise compared to 358
4. 3/5 volt operable.
5. compensated for unity gain

TL072 has similar advantages
1. common
2. 4 MHZ GBP
3. Jfet input
4. low noise
5.  5 volt operable.
6. linear voltage gain  upto 500KHZ with 5volt supply.

**Comment by fhdm-dev** [comment]
I hadn't previously considered the TL072 because I thought the output voltage couldn't swing close enough to the rails but I see the newer version of these chips (the TL07xH family according to the datasheet) can swing to within 100mV - so yeah, that might be worth trying.

**Reply by mausi-mick** [reply]
I use a MCP6022 (Rail-To-Rail/5V supply) and for voltage-protection the dual-diode BAV199 (low inverse current ) on the non inverting input.   For better performance at high frequencies  and lower noise  I make the feedback-resistor with lower resistance ( 10 k between inverting input and output) . On the ADC('s) of the RP2040
I have a voltage divider from 5V to 3.3V (ca 18k and 33k) .

In the schematic the image from the BAV199 is not correct , the BAV199 is in a SOT23 mpackage.

**Comment by mausi-mick** [comment]
![FrontEndRp20240](https://user-images.githubusercontent.com/1472709/163681589-12769f97-c7f2-4357-acd0-86efe18022b4.jpeg)

**Reply by fhdm-dev** [reply]
Thanks for sharing :smile: 
I just tried it with an MCP6282 and 1N4148s (the most similar parts I have on hand) and it works a treat.

![mausi-mick-design](https://user-images.githubusercontent.com/52391579/163761646-758291a9-8a9a-4023-a922-94907f2211cf.jpg)

**Comment by mausi-mick** [comment]
Nice,
perhaps I can test it in the next days with a MCP 653 (single,r-to-r,sot23-6) with better slew-rate !
